operations/README.md

```diff
@@ -14,11 +14,14 @@ All infrastructure operations must be done behind the environment-specific VPN.
 
 Ensure you have the intended git branch checked out and navigate to the `./operations` directory in your CLI. For production deploys, always deploy from the `master` branch.
 
-```
-docker-compose run {dev,test,staging,prod}
-```
+Our Terraform modules are broken out into four stages, each with the dependencies on the previous stages. Each stage can be accessed with an interactive terminal via the following commands:
 
-Running the above command will drop you into an interactive bash terminal for the designated environment.
+```aidl
```

Comment:
Should this be shell instead of aidl?
Also, I would add what the value for `TF_ENV` should be (namely `dev`, `test`, `staging` or `prod`) because those values are what define which backend/tfvars file gets picked up.